javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -109,13 +109,30 @@ DataFlow::Node getThrowTarget(DataFlow::Node thrower) {
  */
 cached
 private module CachedSteps {
+  /** Gets the nesting depth of the given container, starting with the top-level at 0. */
+  cached
+  int getContainerDepth(StmtContainer container) {
```

Comment:
```suggestion
  private int getContainerDepth(StmtContainer container) {
```

It looks like this could be `private`.

Reply:
No, it's referenced on this line in `Configuration.qll`
```codeql
getCapturedVariableDepth(variable) < getContainerDepth(f) and
```

Reply:
Right. I'm not sure why I missed that.